tests/test_codebase/test_mmdet3d/data/smoke_dla34_dlaneck_gn-all_4xb8-6x_kitti-mono3d.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+_base_ = ['kitti-mono3d.py', 'smoke.py', 'default_runtime.py']
```

Comment:
do not have base config:  `default_runtime.py`.

Reply:
default_runtime.py is in main branch. 
https://github.com/open-mmlab/mmdeploy/blob/main/tests/test_codebase/test_mmdet3d/data/default_runtime.py